Backend/main.py

```diff
@@ -2,20 +2,23 @@
 Starting file for uvicorn web server.
 Containing an app object to start from.
 """
-from fastapi import FastAPI
 import asyncio
+
+from fastapi import FastAPI
 from fastapi.middleware.cors import CORSMiddleware
 from src.routers import *
 from src.util.receiver import *
 from src.util.sender import *
 from src.auth.jwt_handler import AuthorizeMiddleware
+import time
 
 app = FastAPI()
 
 
 @app.on_event('startup')
 async def startup() -> None:
     threading.Thread(target=asyncio.run, args=(test(),)).start()
+    time.sleep(5)

```

Comment:
Das `sleep` wird es auch nicht besser machen.
Ich glaube das prinzip der globalen variablen in python über mehrere Dateien ist einfach schwierig.
Du könntest es eher nochmal mit `os.environ["MY_ENV"] = "VALUE"` probieren...